app/controllers/comments_controller.rb

```diff
@@ -50,21 +53,21 @@ def update
   # DELETE /comments/1 or /comments/1.json
   def destroy
     @comment.destroy
-
     respond_to do |format|
       format.html { redirect_to comments_url, notice: "Comment was successfully destroyed." }
       format.json { head :no_content }
     end
   end
 
   private
-    # Use callbacks to share common setup or constraints between actions.
-    def set_comment
-      @comment = Comment.find(params[:id])
-    end
 
-    # Only allow a list of trusted parameters through.
-    def comment_params
-      params.require(:comment).permit(:author_id, :photo_id, :body)
-    end
+  # Use callbacks to share common setup or constraints between actions.
+  def set_comment
+    @comment = Comment.find(params[:id])
+  end
+
+  # Only allow a list of trusted parameters through.
+  def comment_params
+    params.require(:comment).permit(:photo_id, :body)
```

Comment:
I would recommend adding `:author_id` as well.

Reply:
thanks added it